packages/@aws-cdk/custom-resource-handlers/lib/aws-route53/cross-account-zone-delegation-handler/index.ts

```diff
@@ -15,6 +15,7 @@ interface ResourceProperties {
   DelegatedZoneName: string,
   DelegatedZoneNameServers: string[],
   TTL: number,
+  STSRegion?: string,
```

Comment:
Why `STSRegion`? 

I like to be able to deduce what a property does from the name so not sure if `STSRegion` is the right (could just be that this makes sense for people familiar with cross account zone delegation, in which case I'd be happy with keeping it as `STSRegion`).

Reply:
Renamed to `AssumeRoleRegion`